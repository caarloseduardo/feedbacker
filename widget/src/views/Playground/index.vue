<template>
  <div class="w-full h-3/4 flex flex-col justify-center items-center bg-brand-main">
    <h1 class="text-6xl font-black text-brand-gray">Playground</h1>
    <p class="text-2xl mt-3 font-regular text-brand-gray">Este Ã© o playground, use para testar o widget.</p>
  </div>
  <div class="w-full h-3/4 flex flex-col justify-center items-center bg-brand-gray">
    <h1 class="text-6xl font-black text-brand-graydark">teste1</h1>
  </div>
  <div class="w-full h-3/4 flex flex-col justify-center items-center bg-brand-main">
    <h1 class="text-6xl font-black text-brand-graydark">teste2</h1>
  </div>
  <div class="w-full h-3/4 flex flex-col justify-center items-center bg-brand-gray">
    <h1 class="text-6xl font-black text-brand-graydark">teste3</h1>
  </div>
  <widget />
</template>

<script lang="ts">
import { defineComponent } from 'vue'
import Widget from '@/views/Widget/index.vue'

export default defineComponent({
  components: { Widget }
})
</script>
