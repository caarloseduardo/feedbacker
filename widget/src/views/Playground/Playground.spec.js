import { shallowMount } from '@vue/test-utils'
import Playground from './index.vue'

describe('<Playground />', () => {
  it('should component render correctly', () => {
    const wrapper = shallowMount(Playground)
    expect(wrapper).toMatchSnapshot()
  })
})
