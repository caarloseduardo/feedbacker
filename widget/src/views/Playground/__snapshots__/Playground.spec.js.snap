// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Playground /> should component render correctly 1`] = `
VueWrapper {
  "__app": Object {
    "_component": Object {
      "__emits": Object {},
      "__props": Array [
        Object {},
        Array [],
      ],
      "name": "VTU_ROOT",
      "render": [Function],
    },
    "_container": <div
      data-v-app=""
    >
      
      <div
        class="w-full h-3/4 flex flex-col justify-center items-center bg-brand-main"
      >
        <h1
          class="text-6xl font-black text-brand-gray"
        >
          Playground
        </h1>
        <p
          class="text-2xl mt-3 font-regular text-brand-gray"
        >
          Este Ã© o playground, use para testar o widget.
        </p>
      </div>
      <div
        class="w-full h-3/4 flex flex-col justify-center items-center bg-brand-gray"
      >
        <h1
          class="text-6xl font-black text-brand-graydark"
        >
          teste1
        </h1>
      </div>
      <div
        class="w-full h-3/4 flex flex-col justify-center items-center bg-brand-main"
      >
        <h1
          class="text-6xl font-black text-brand-graydark"
        >
          teste2
        </h1>
      </div>
      <div
        class="w-full h-3/4 flex flex-col justify-center items-center bg-brand-gray"
      >
        <h1
          class="text-6xl font-black text-brand-graydark"
        >
          teste3
        </h1>
      </div>
      <widget-stub />
      
    </div>,
    "_context": Object {
      "app": [Circular],
      "components": Object {
        "transition": Object {
          "name": "transition",
          "props": undefined,
          "render": [Function],
        },
        "transition-group": Object {
          "name": "transition-group",
          "props": undefined,
          "render": [Function],
        },
      },
      "config": Object {
        "errorHandler": undefined,
        "globalProperties": Object {},
        "isCustomElement": [Function],
        "isNativeTag": [Function],
        "optionMergeStrategies": Object {},
        "performance": false,
        "warnHandler": undefined,
      },
      "directives": Object {},
      "mixins": Array [
        Object {
          "__emits": null,
          "__props": Array [],
          "beforeCreate": [Function],
        },
      ],
      "provides": Object {},
      "reload": [Function],
    },
    "_props": null,
    "_uid": 0,
    "component": [Function],
    "config": Object {
      "errorHandler": undefined,
      "globalProperties": Object {},
      "isCustomElement": [Function],
      "isNativeTag": [Function],
      "optionMergeStrategies": Object {},
      "performance": false,
      "warnHandler": undefined,
    },
    "directive": [Function],
    "mixin": [Function],
    "mount": [Function],
    "provide": [Function],
    "unmount": [Function],
    "use": [Function],
    "version": "3.0.5",
  },
  "__setProps": [Function],
  "componentVM": Object {},
  "rootVM": Object {},
}
`;
